src/backend.ts

```diff
@@ -311,14 +321,52 @@ export function setupSuspendResumeHandler(): () => void {
 export function setupFocusChangeHandler(): () => void {
   let appIsStartingUp: boolean = false;
   let lastPid = 0;
+  let lastAppid = 0;
+  let validKeyEvent: SystemKeyEvent | null = null;
+  SteamClient.System.UI.RegisterForSystemKeyEvents((e) => {
```

Comment:
It's unfortunate that this doesn't seem to return an unregister function. Maybe to alleviate that a bit, the inner logic could be wrapped in another function that can be set to the noop (`() => {}`) on dismount.

Reply:
Added:

```ts
let keyEventFunction = (e: SystemKeyEvent) => {
    // Have not found any race condition issues with this approach since the key event fires long before focus change
    validKeyEvent = e;
  }
  const unregisterSystemKeyEvents = () => {
    keyEventFunction = () => {};
  }
  SteamClient.System.UI.RegisterForSystemKeyEvents((e) => keyEventFunction(e));
```